### PR TITLE
Update directory-currency.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Directory Currency GraphQL with Multiple currency as the response. How Base currency and other currency of the store will be response of the query.

## Purpose of this pull request
An Example for the multi currency store such as USD and EURO. Default Example given in the dev docs is for the single currency only. I have given an example with two currency for the website. Check with GraphQL response and apply solutions.

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/graphql/queries/directory-currency.html
https://devdocs.magento.com/guides/v2.4/graphql/queries/directory-currency.html

### CODE REVIEW

File: src/guides/v2.3/graphql/queries/directory-currency.md [hunk 1]
```diff
@@ -55,6 +55,55 @@ query {
 }
 ```
 
+The following query returns the currency for the current instance of Magento with multiple currency such as USD and EURO. Default(Base) Currency for the store is US Dollar(USD). GraphQL query will be same but response will be different based on available currency. List of arrays of the currency will be displayed in available_currency_codes attribute while exchange_rates attribute contains the array of currency_to and rate vaule for respective currency.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/directory-currency.md:58]:
```suggestion
The following query returns the currency for the current instance of Magento with multiple currencies such as USD and EURO. The default(base) currency for the store is US Dollar(USD). The GraphQL query will be same , but the response will be different based on the available currency. A list of arrays of the currency will be displayed in the `available_currency_codes` attribute while the `exchange_rates` attribute contains the array of `currency_to` and rate value for the respective currency.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please update PR description to the required format and information. A good example : https://github.com/magento/devdocs/pull/8296

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have updated the Comment and PR. Thank you for your suggestion.

File: src/guides/v2.3/graphql/queries/directory-currency.md [hunk 2]
```diff
@@ -55,6 +55,55 @@ query {
 }
 ```
 
+The following query returns the currency for the current instance of Magento with multiple currencies such as USD and EURO. The default(base) currency for the store is US Dollar(USD). The GraphQL query will be same , but the response will be different based on the available currency. A list of arrays of the currency will be displayed in the `available_currency_codes` attribute while the `exchange_rates` attribute contains the array of `currency_to` and rate value for the respective currency.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/queries/directory-currency.md:58]:
```suggestion
The following query returns the currency for the current instance of Magento with multiple currencies such as USD and EURO. The default(base) currency for the store is US Dollar(USD). The GraphQL query will be same, but the response will be different based on the available currency. A list of arrays of the currency will be displayed in the `available_currency_codes` attribute while the `exchange_rates` attribute contains the array of `currency_to` and rate value for the respective currency.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Not sure how that crept in... All good after

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have update the code based on your suggestion. Thanks.

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
I deleted the original example, because there's no need to show one query doing the same thing twice, and your example was better.

**Pull request merged**